.github/workflows/nodejs.yml

```
@@ -2,38 +2,42 @@ name: Node.js CI
 
 on:
   push:
-    branches: master
+    branches:
+      - $default-branch
```

```suggestion
      - master
```

default-branch is a placeholder that github replaces when you create the workflow from the Actions tab